server/src/validation.ts

```diff
@@ -1,4 +1,7 @@
 import * as yup from 'yup';
+import { createError } from '@typescript-eslint/typescript-estree/dist/node-utils';
```

Comment:
Should this be from the dist of this lib?

Reply:
It shouldn't be. removed. do we need to add a rule to check unused imports?

Reply:
Sure. pls update eslint as required.